### PR TITLE
Better integration with GraphQL CLI

### PR DESCRIPTION
**Pull request by kamilkisiela**

### CODE REVIEW

File: packages/graphql-cli-codegen/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@graphql-cli/codegen",
```
Comment (dotansimha) [inline packages/graphql-cli-codegen/package.json:2]: Maybe just change the directory to be `graphql-cli-codegen-plugin` and `cli` ? it's a bit confusing now :D

Comment (kamilkisiela) [reply packages/graphql-cli-codegen/package.json:2]: sure

Comment (kamilkisiela) [reply packages/graphql-cli-codegen/package.json]: @dotansimha renamed

File: packages/graphql-cli-codegen-plugin/package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  "description": "GraphQL Code Generator's GraphQL CLI plugin. GraphQL Code Generator is a tool that generates code from your GraphQL schema and documents for your backend or frontend with flexible support for custom plugins and templates.",
+  "license": "MIT",
+  "keywords": [
+    "graphql-cli",
+    "graphql-cli-plugin"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dotansimha/graphql-code-generator.git",
+    "directory": "packages/graphql-cli-codegen"
```
Comment (kamilkisiela) [inline packages/graphql-cli-codegen-plugin/package.json:13]:
```suggestion
    "directory": "packages/graphql-cli-codegen-plugin"
```

**Pull request merged**